api/src/main/java/cz/muni/fi/pa165/dto/BottleTypeDTO.java

```diff
@@ -10,10 +13,13 @@
 public class BottleTypeDTO {
     private Long id;
 
+    @NotNull
     private String name;
 
+    @DecimalMin("0")
     private BigDecimal volume;
 
+    @DecimalMin("0.5")
```

Comment:
Je to mozno privela, ale priznajme si ze na slovensku a v cechach ludia piju dost a nikto nekupuje menej ako pol litra alkoholu :D